billy/web/api/handlers.py

```diff
@@ -469,9 +481,27 @@ def read(self, request, abbr, chamber=None):
 class BoundaryHandler(BillyHandler):
     base_url = settings.BOUNDARY_SERVICE_URL
 
+    def _ocd_id_to_shape_url(self, ocd_id):
```

Comment:
This is a bit of a pickle. This means we're hitting the BS twice for each call.

I think what this means is we need a way of getting the "current" geometry back from the service *without* making two calls. Right now, we can eat this traffic, since it's not huge, but I hate we're dealing with an extra RTT on HTTP